util.js

```diff
@@ -5,7 +5,7 @@ var path = require("path");
 var mainFiles = ["ca-gregorian", "currencies", "dateFields", "numbers", "timeZoneNames", "units"];
 
 var isGlobalizeModule = function(filepath) {
-  filepath = filepath.split( "/" );
+  filepath = filepath.split( /[\/\\]/ );
```

Comment:
You can use `path.sep` in node to get the platform specific file separator (a bit of a pain with some of the regex literals in this diff though).